tests/Contacts/PhoneNumberTest.m

```diff
@@ -59,4 +78,72 @@ - (void)testTryParsePhoneNumberFromUserSpecifiedTextRemovesAnyFormatting {
     XCTAssertEqualObjects(@"+13235551234", [actual toE164]);
 }
 
+- (NSArray<NSString *> *)unpackTryParsePhoneNumbersFromsUserSpecifiedText:(NSString *)text
+                                                        clientPhoneNumber:(NSString *)clientPhoneNumber
+{
+    NSArray<PhoneNumber *> *phoneNumbers =
+        [PhoneNumber tryParsePhoneNumbersFromsUserSpecifiedText:text clientPhoneNumber:clientPhoneNumber];
+    NSMutableArray<NSString *> *result = [NSMutableArray new];
+    for (PhoneNumber *phoneNumber in phoneNumbers) {
+        [result addObject:phoneNumber.toE164];
+    }
+    return result;
+}
+
+- (void)testTryParsePhoneNumbersFromsUserSpecifiedText_SimpleUSA
+{
+    NSArray<NSString *> *parsed =
+        [self unpackTryParsePhoneNumbersFromsUserSpecifiedText:@"323 555 1234" clientPhoneNumber:@"+13213214321"];
+    XCTAssertLessThanOrEqual(1, parsed.count);
```

Comment:
I find some of the XCT macros to be a bit hard to read - the compare functions among them. I guess I'm just used to using comparators in infix notation.

So, I think this is equivalent to `XCTAssert(1 <= parsed.count)`
But I think it'd be a little more idiomaticly expressed in reverse:  `XCTAssert(parsed.count >= 1)`
Which I guess would be: `XCTAssertGreaterThanOrEqual(parsed.count, 1)`
But honestly I prefer the: `XCTAssert(parsed.count >= 1)`

I know it's all the same, so feel free to ignore, but I had to do a triple take when reading this.